src/renderer/components/tree-view/tree-view.tsx

```diff
@@ -0,0 +1,102 @@
+import "./tree-view.scss";
+
+import React, { useEffect, useRef } from "react";
+import { Icon } from "../icon";
+import TreeView from "@material-ui/lab/TreeView";
+import TreeItem from "@material-ui/lab/TreeItem";
+import { cssNames } from "../../utils";
+
+import _ from "lodash";
+import getDeepDash from "deepdash";
+
+const deepDash = getDeepDash(_);
+
+export interface NavigationTree {
+  id: string;
+  parentId: string;
+  name: string;
+  selected?: boolean;
+  children?: NavigationTree[];
+}
+
+interface Props {
+  data: NavigationTree[]
+}
+
+const scrollToItem = (id: string) => {
+  const element = document.getElementById(id);
+
+  element?.scrollIntoView();
+};
```

Comment:
This should be a function
```suggestion
function scrollToItem(id: string): void {
  document
  	.getElementById(id)
    ?.scrollIntoView();
}
```